frontendActions/build.gradle

```diff
@@ -2,6 +2,7 @@ dependencies {
   vavr()
 
   implementation project(':backendApi')
+  implementation project(':branchLayoutImpl')
```

Comment:
TBD

Reply:
Maybe use the trick from `backendRoot`? Compile-time dependency on `branchLayoutApi` + loading the factory for an implementing class instance in runtime via `Reflections`